18 Normalizr/src/reducers/domain/student/index.ts

```diff
@@ -2,12 +2,18 @@ import { combineReducers } from 'redux';
 import { byId, allIds} from './list';
 import { edit } from './edit'
 import { StudentView } from '../../../model/view/studentView';
+import { getCountry } from '../country/byId';
 
 export const studentDomain = combineReducers({
   byId,
   allIds,
   edit
 });
 
-export const getStudent = (state, id) : StudentView => state[id];
+export const getStudent = (state, id) : StudentView => {
+  return {
+    ...state.studentDomain.byId[id],
+    country: getCountry(state, state.studentDomain.byId[id].country)
```

Comment:
I got lost: previously state was the dictionary of students. Now it's the global state. This seems hard to manage unless you strong-type arguments to prevent mismatches.